attoparsec-iso8601/src/Data/Attoparsec/Time.hs

```diff
@@ -72,6 +82,17 @@ quarter = do
       <|> Q3 <$ char '3'
       <|> Q4 <$ char '4'
 
+-- | Parse a year @YYYY@, with at least 4 digits. Does not include any sign.
```

Comment:
It might be worth noting that `Year` is just a `type Year = Integer`, not `newtype`. And in relatively recent `time` (`1.11`). `time-compat`  defines such an alias for older `time`s too.